app/models/info_request.rb

```diff
@@ -39,7 +39,7 @@ class InfoRequest < ActiveRecord::Base
   include Rails.application.routes.url_helpers
 
   # Two sorts of laws for requests, FOI or EIR
-  LAW_USED_READABLE_DATA =
+  def law_used_readable_data
```

Comment:
should this be a private method? I think we can still override it in themes if it is